distributed/stealing.py

```diff
@@ -40,8 +40,14 @@ def __init__(self, scheduler):
         for worker in scheduler.workers:
             self.add_worker(worker=worker)
 
+        callback_time = parse_timedelta(
+            dask.config.get("distributed.scheduler.work-stealing-interval"),
+            default="ms",
```

Comment:
I think you'll need something like "100ms" here.

Reply:
AFAI see, `default` represents the default time unit (`def parse_timedelta(s, default="seconds")`)

Reply:
Oh, sorry I misunderstood.